docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -68,23 +68,23 @@ autonomy mint skill [OPTIONS] PACKAGE_PATH
 : Dependencies for the package.
 
 `--nft IPFS_HASH_OR_IMAGE_PATH`
-: IPFS hash or path to the NFT image for the NFT image representing the package. Note that if you are using a local chain this option is not required.
+: IPFS hash or path to the image for the NFT representing the package. Note that if you are using a local chain this option is not required.
 
-`--owner TEXT`
+`--owner OWNER_ADDRESS`
 : Owner address of the package.
 
 ### Examples
 
-Mint the `hello_world_abci` {{fsm_app}} skill with dependencies 11 and 42 in a custom chain:
+Mint the `abstract_abci` skill with dependency IDs 35 and 42 in a local chain:
 
 ```bash
-autonomy mint --use-custom-chain skill -d 11 -d 42 --nft <nft_ipfs_hash_or_image_path> ./packages/valory/skills/hello_world_abci --key my_key.txt
+autonomy mint --use-local skill --key my_key.txt --nft <nft_ipfs_hash_or_image_path> --owner <owner_address> -d 35 -d 42 ./packages/valory/skills/abstract_abci
 ```
 
 Same as above, but using a hardware wallet:
 
 ```bash
-autonomy mint --use-custom-chain skill -d 11 -d 42 --nft <nft_ipfs_hash_or_image_path> ./packages/valory/skills/hello_world_abci --hwi
+autonomy mint --use-local skill --hwi --nft <nft_ipfs_hash_or_image_path> --owner <owner_address> -d 35 -d 42 ./packages/valory/skills/abstract_abci
```

Comment:
This is wrong, this tutorial makes an assumption that the chain has already has the dependencies published which is true in case of a local mint performed against `valory/autonolas-registries` image, but if this changes this tutorial won't work. And this will also confuse the users since the dependency IDs are explicitly defined and users can try to use the same dependencies when minting against testnet or mainnet.

This examples needs to generalised and chain agnostic.

Reply:
Note that the actual tutorial tailor made for the autonolas-registries is here https://github.com/valory-xyz/open-autonomy/pull/1936/files#diff-2a94735d14f317084aed3d0590fe0ee3c43a32f09a93e1c71396d4d0c6e74b97R157-R189

And a warning to the user pointing exactly the issue you raise is written here: 
https://github.com/valory-xyz/open-autonomy/pull/1936/files#diff-2a94735d14f317084aed3d0590fe0ee3c43a32f09a93e1c71396d4d0c6e74b97R149-R155